scripts/qa_crowbarsetup.sh

```diff
@@ -1332,6 +1332,8 @@ function set_node_raid
     disks_count="$3"
 
     wait_for 10 5 "getent hosts $node &> /dev/null" "$node name to be resolvable"
+
+    wait_for 300 10 "timeout -k 20 10 ssh -o UserKnownHostsFile=/dev/null $node true" "$node to become sshable"
```

Comment:
Why not `wait_for` using a netcat on port 22? (e.g. https://github.com/SUSE-Cloud/automation/blob/df48fba89925f4ddbce56db36d0e4fbe21db8b4a/scripts/qa_crowbarsetup.sh#L3587)